.github/workflows/python_pytest.yml

```diff
@@ -39,6 +39,9 @@ jobs:
     - name: Install dependencies
       run: poetry install
 
+    - name: Create test results directory
+      run: mkdir -p build/test-results
+
```

Comment:
Is this necessary (pre-creating the directory)? I thought PyTest would create it automatically if missing.